playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -51,6 +51,9 @@
       '{"spec":{"template":{"spec":{"containers":[{"name":"router","image":"{{ router_image }}","livenessProbe":{"tcpSocket":null,"httpGet":{"path": "/healthz", "port": 1936, "host": "localhost", "scheme": "HTTP"},"initialDelaySeconds":10,"timeoutSeconds":1}}]}}}}'
       --api-version=v1
     with_items: "{{ haproxy_routers }}"
+    # AUDIT:changed_when_note: `false` not being set here. What we
```

Comment:
As long as we're going to sweep all AUDIT: in a future sprint I'm fine with leaving this, but if we're going to leave some in as notes to indicate that we believe the changed_when: false to be benign I think we should add TODO to anything where we're suggesting future fixes.